digiwf-email-integration-starter/src/main/java/io/muenchendigital/digiwf/email/integration/configuration/CustomMailProperties.java

```diff
@@ -0,0 +1,12 @@
+package io.muenchendigital.digiwf.email.integration.configuration;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Data
+@ConfigurationProperties(prefix = "digiwf.mail")
+public class CustomMailProperties {
+
+    private String fromAdress = "noreply@noreply.com";
```

Comment:
`fromAddress` d is missing
would add a @NotBlank and no default value

Reply:
As far as I know, you only need fromAdress when not using smptAuth though. That's why I made it kind of optional. You still think it's better to use @NotBlank?